source/features/keyboard-navigation.tsx

```diff
@@ -0,0 +1,53 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+function runShortcuts(event: KeyboardEvent): void {
+	if (isEditable(event.target)) {
+		return;
+	}
+
+	const focusedComment = select(':target')!;
+
+	if (['j', 'k'].includes(event.key)) {
+		event.preventDefault();
+
+		const items = select.all('.js-targetable-element')
+			.filter(comment => !comment.querySelector('.minimized-comment:not(.d-none)'));
+		// `j` goes to the next comment, `k` goes back a comment
+		const direction = event.key === 'j' ? 1 : -1;
```

Comment:
This should be the opposite. `j` should be previous and `k` should be next.

```suggestion
		const direction = event.key === 'j' ? -1 : 1;
```


Also this looks very slow, especially on large PRs (not ou  r fault though, its GH).

And pressing the same key repeatedly (on Firefox) fails to take us to the right comment (GHs `window.scrollTo()` call for some reason gets delayed and adds up eventually scrolling to random positions).

Reply:
I am following GitHub

![image](https://user-images.githubusercontent.com/16872793/86354849-c1457800-bc37-11ea-8ca9-549125f53003.png)

Reply:
> adds up eventually scrolling to random positions

This is probably GitHub screwing it up: https://github.com/sindresorhus/refined-github/issues/3263